Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG TARGETARCH
 ARG TARGETOS
 
-FROM us.gcr.io/k8s-artifacts-prod/build-image/debian-base-amd64:v2.1.0
```

Comment:
I think new format for these is `buster-v1.2.0`

https://github.com/kubernetes/release/pull/1496

Reply:
Ah, so they are tagging both -
```yaml
    "sha256:dc06e242160076b72bd75135fb3dd0a9e91f386b2d812ec10cbf9e65864c755d": ["buster-v1.1.3", "v2.1.3"]
```

Reply:
Let me update it to use the new format

Reply:
do we want `1.1.3` or `1.2.0`?

Reply:
`1.1.3` is `v2.1.3`. Looks like `1.2.0` was promoted 3hrs ago - https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/images/k8s-staging-build-image/images.yaml#L14. We can go to `1.2.0`.

Reply:
Updated to `buster-v1.2.0`